bootstrap.py

```diff
@@ -766,6 +767,40 @@ class MEOptions:
     disable_rhn_plugin()
 
 
+def enable_service(service, failonerror=True):
+    """
+    Helper function to enable a service using proper init system.
+    pass failonerror = False to make init system's commands non-fatal
+    """
+    if failonerror:
+        if os.path.exists("/run/systemd"):
+            exec_failexit("/usr/bin/systemctl enable %s" % (service))
+        else:
+            exec_failexit("/sbin/chkconfig %s on" % (service))
+    else:
+        if os.path.exists("/run/systemd"):
+            exec_failok("/usr/bin/systemctl enable %s" % (service))
+        else:
+            exec_failok("/sbin/chkconfig %s on" % (service))
+
+
+def start_service(service, command="start", failonerror=True):
```

Comment:
I like the way you made it possible to start/restart/stop the service. Shall we call the wrapper just `service`, as it can do more than start?